.github/workflows/build-test-centos7.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     - run: sh autogen.sh
     - run: ./configure
     - run: make distcheck
-
```

Comment:
Hi Karin, this is an empty change. It just adds white space. This is a nit, but can you please remove this white space, there should be no *.yaml files in your change list.

Reply:
Sure, they are removed.